flexmeasures/ui/static/js/daterange-utils.js

```diff
@@ -38,4 +38,33 @@ function getTimeZoneOffset(date, timeZone) {
     // Positive values are West of GMT, opposite of ISO 8601
     // this matches the output of `Date.getTimeZoneOffset`
     return -(lie - date) / 60 / 1000;
-}
\ No newline at end of file
+}
+
+/**
+ * Count the number of Daylight Saving Time (DST) transitions within a given datetime range.
+ * @param {Date} startDate - The start date of the datetime range.
+ * @param {Date} endDate - The end date of the datetime range.
+ * @param {number} increment - The number of days to increment between iterations.
+ * @returns {number} The count of DST transitions within the specified range.
+ */
+export function countDSTTransitions(startDate, endDate, increment) {
```

Comment:
Just to confirm, `startDate` will never be equal to `endDate` , for example, when selecting just a day?

Reply:
Correct.